domains/onichan.json

```diff
@@ -6,6 +6,6 @@
     "email": "rbstsoul@gmail.com"
   },
   "record": {
-    "CNAME": "hosting.gitbook.io"
+    "CNAME": "onichan.readthedocs.io"
```

Comment:
do you really want to use this cname?

iirc readthedocs cname is `readthedocs.io` for custom domain.

Reply:
let me fix it

Reply:
did it